cluster/images/conformance/Makefile

```diff
@@ -18,20 +18,52 @@
 #   [ARCH=amd64] [REGISTRY="k8s.gcr.io"] make (build|push) VERSION={some_released_version_of_kubernetes}
 
 REGISTRY?=k8s.gcr.io
+OS?=linux
+OS_VERSION?=1809
 ARCH?=amd64
 OUT_DIR?=_output
 
-LOCAL_OUTPUT_PATH=$(shell pwd)/../../../$(OUT_DIR)/local/bin/linux/$(ARCH)
-DOCKERIZED_OUTPUT_PATH=$(shell pwd)/../../../$(OUT_DIR)/dockerized/bin/linux/$(ARCH)
+# The output type could either be docker (local), or registry.
+# If it is registry, it will also allow us to push the Windows images.
+OUTPUT_TYPE = docker
+ifeq ($(MAKECMDGOALS),push)
+  OUTPUT_TYPE = registry
+else ifeq ($(OS),windows)
+    $(error Windows images can only be created and pushed directly. Use make push for Windows images)
```

Comment:
You can also output to a tarball and then use skopeo to optionally push it. Thats what we did for Sonobuoy but I'm not sure if k8s uses that tool elsewhere. Using push is for windows is probably fine though if it works for peoples usecases.